src/Function/Relation/Binary/Setoid/Equality.agda

```diff
@@ -0,0 +1,50 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Function Equality setoid
+------------------------------------------------------------------------
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+open import Level using (Level; _⊔_)
+open import Relation.Binary.Bundles using (Setoid)
+
+module Function.Relation.Binary.Setoid.Equality {a₁ a₂ b₁ b₂ : Level}
+  (From : Setoid a₁ a₂) (To : Setoid b₁ b₂) where
+
+  open import Function.Bundles using (Func; _⟨$⟩_)
+  open import Relation.Binary.Definitions
+    using (Reflexive; Symmetric; Transitive)
+    
+  private
+    module To = Setoid To
+    module From = Setoid From
+
+  infix 4 _≈_
+  _≈_ : (f g : Func From To) → Set (a₁ ⊔ b₂)
+  f ≈ g = {x : From.Carrier} → f ⟨$⟩ x To.≈ g ⟨$⟩ x
```

Comment:
We could get away with removing thistype annotation if we used a forall?

Reply:
We could, but I rather think that this one serves as useful documentation.

Reply:
Okay sure.

Reply:
Any reason to favour the 'diagonal' version of the relation, rather than the PER-like
```agda
f ≈ g = {x y : From.Carrier} → x From.≈ y → f ⟨$⟩ x To.≈ g ⟨$⟩ y
```
(which, incidentally/moreover, would also permit use of a forall...)

Reply:
I forget the exact places where the discussion was had, but this is an explicit choice. The seeming extra generality doesn't buy you anything as these two are inter-derivable and it seems that in practice, it is this version which is more often needed.

Certainly we went back-and-forth on this in `agda-categories` and the version here definitely makes things simpler, at no loss of generality.

Reply:
Yes, I now see the discussion on #1467 and while I may *personally* disagree, ... whatever!?